### PR TITLE
Cuda101 upgrade

### PR DESCRIPTION
**Pull request by mingbowan**
test run: #26732

### CODE REVIEW

File: .circleci/verbatim-sources/binary-build-params.yml [hunk 1]
```diff
@@ -15,6 +15,7 @@ binary_linux_build_params: &binary_linux_build_params
   environment:
     BUILD_ENVIRONMENT: << parameters.build_environment >>
     LIBTORCH_VARIANT: << parameters.libtorch_variant >>
+    ANACONDA_USER: pytorch
```
Comment (ezyang) [inline .circleci/verbatim-sources/binary-build-params.yml]: Hmm?

**Comment by ezyang** [comment]
The submodule updates are probably wrong

**Comment by ezyang** [comment]
Can we drop 10.0 builds with this?

**Review by soumith — APPROVED** [review]
remove submodule changes before landing

File: .circleci/verbatim-sources/binary-build-params.yml (hunk 1 above)
Comment (mingbowan) [reply .circleci/verbatim-sources/binary-build-params.yml]: default user is soumith which is the conda channel name, but channel soumith is missing dependency to build cuda 10.1. @soumith pointed out that correct user we should use is "pytorch"

**Pull request closed without merging**